ui/component/or-attribute-history/src/index.ts

```diff
@@ -381,90 +396,136 @@ export class OrAttributeHistory extends translate(i18next)(LitElement) {
                     }
                 }
 
-                const options = {
-                    type: "line",
-                    data: {
-                        datasets: [
-                            {
-                                data: data,
-                                backgroundColor: bgColor,
-                                borderColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-line-color"),
-                                pointBorderColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-border-color"),
-                                pointBackgroundColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-color"),
-                                pointRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-radius")),
-                                pointBorderWidth: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-border-width")),
-                                pointHoverBackgroundColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-color"),
-                                pointHoverBorderColor: this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-border-color"),
-                                pointHoverRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-radius")),
-                                pointHoverBorderWidth: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hover-border-width")),
-                                pointHitRadius: Number(this._style.getPropertyValue("--internal-or-attribute-history-graph-point-hit-radius")),
-                                fill: false
+                this._chartOptions = {
+                    animation: false,
+                    grid: {
+                        show: true,
+                        backgroundColor: this._style.getPropertyValue("--internal-or-asset-viewer-panel-color"),
+                        borderColor: this._style.getPropertyValue("--internal-or-attribute-history-text-color"),
+                        left: 15,
+                        right: 15,
```

Comment:
*(If we remove the tooltip as mentioned in the comment on line 425, we can remove the left-right margin)*
```suggestion
                        left: 0,
                        right: 4,
```

Reply:
yep, adjusted to 4,4 now, would cut some of the y axislabels text for large numbers otherwise.